src/python_testing/TC_RVCOPSTATE_2_5.py

```diff
@@ -19,7 +19,7 @@
 # test-runner-runs:
 #   run1:
 #     app: ${ALL_CLUSTERS_APP}
```

Comment:
should use RVC app maybe like the other RVCOPSTATE tests?

In any case, I rolled this back in #39997